core/src/main/java/com/linecorp/armeria/internal/client/ClientThreadLocalState.java

```diff
@@ -22,21 +22,23 @@
 
 import com.google.common.collect.ImmutableList;
 
+import com.linecorp.armeria.client.ClientRequestContext;
+import com.linecorp.armeria.client.ClientRequestContextCaptor;
 import com.linecorp.armeria.common.annotation.Nullable;
 import com.linecorp.armeria.common.util.SafeCloseable;
 
 import io.netty.util.concurrent.FastThreadLocal;
 
-final class ClientThreadLocalState {
+public final class ClientThreadLocalState {
```

Comment:
Could we add a private constructor to prevent this class from being created externally?

Reply:
Thanks! Done